README.md

```diff
@@ -24,13 +24,25 @@ The easiest way to get started locally is to simply [get yourself a copy of open
 $ java -jar joshsim.jar run simulation.josh
 ```
 
-While COGs, geotiffs, and netCDF files can be provided directly, the preferred approach is to provide a jshd file which preprocesses these geospatial inputs for speed:
+While COGs, geotiffs, and netCDF files can be provided directly, the preferred approach is to provide a jshd file which preprocesses these geospatial inputs for speed. These jshd files are loaded from the working directory or included with the request:
 
 ```
 $ java -jar joshsim.jar preprocess simulation.josh MySimulation data.nc variable units output.jshd
 $ java -jar joshsim.jar run simulation.josh --data output.jshd
```

Comment:
The data flag is not yet available? If not, let's leave this out of the example but maybe we can add a prior line:

```
$ ls output.jshd
```